packages/ui-extensions/src/surfaces/customer-account/api/shared.ts

```diff
@@ -1162,3 +1162,43 @@ export interface MailingAddress {
    */
   phone?: string;
 }
+
+export interface Account {
+  /**
+   * Provides the company info of the logged in business customer.
+   * If the customer is not logged in or is not a business customer, this value is `undefined`.
+   */
+  purchasingCompany: StatefulRemoteSubscribable<PurchasingCompany | undefined>;
+  /**
+   * Provides the customer information of the logged in customer.
+   */
+  customer: StatefulRemoteSubscribable<Customer | undefined>;
+}
+
+/**
+ * Information about the customer who is viewing the page.
+ *
+ * {% include /apps/checkout/privacy-icon.md %} Requires access to [protected customer data](/docs/apps/store/data-protection/protected-customer-data).
+ */
+export interface Customer {
+  /**
+   * Customer ID.
+   *
+   * @example 'gid://shopify/Customer/123'
+   */
+  id: string;
+}
+
+export interface PurchasingCompany {
```

Comment:
Should this be part of a shared interface with the purchasing company on order status ?